recipes/shtns/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "shtns" %}
+{% set version = "3.3.1" %}
+{% set revision = "r694" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://www.bitbucket.org/nschaeff/{{ name }}/downloads/{{ name }}-{{ version }}-{{ revision }}.tar.gz
+    sha256: 3fcca0ffcf28467fd80ef71e0032c15ac6deac5c592041dc6a9a7f4575fbe64c
+    folder: src
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+
+  build:
+    - m2-bash  # [win]
+    - m2-curl  # [win]
+    - m2-findutils  # [win]
+    - m2-diffutils  # [win]
+    - m2-gawk  # [win]
+    - m2-grep  # [win]
+    - m2-make  # [win]
+    - m2-make  # [win]
+    - m2-sed  # [win]
+    - m2-which  # [win]
+    - m2-tar  # [win]
+    - swig  # [win]
+    - {{ compiler('c') }}  # [unix]
+    - {{ compiler('m2w64_c') }}  # [win]
+  host:
+    - fftw
+    - m2w64-binutils  # [win]
+    - m2w64-gcc  # [win]
+    - numpy >=1.12
```

Comment:
please see  https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy for numpy

Reply:
I believe this is now fixed. Thanks for pointing this out.